integration-tests/features/stack_analyses_v2_response_check.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Smoketests for stack analysis V2 API tests
+
+
+  Scenario Outline: Check the stack analysis response for selected input
+    Given System is running
+    Given Three scale preview service is running
+    When I acquire the use_key for 3scale
+    Then I should get the proper user_key
+
+    # request the stack analysis
+    When I wait 10 seconds
+    When I send <ecosystem> package request with manifest <file> to stack analysis v2 with valid user key
+    Then I should get 200 status code
+     And I should receive JSON response containing the status key
+     And I should receive JSON response containing the id key
+     And I should receive JSON response containing the submitted_at key
+     And I should receive JSON response with the status key set to success
+     And I should receive JSON response with the correct id
+     And I should receive JSON response with the correct timestamp in attribute submitted_at
+
+    # Wait for response from stack analysis
+    When I wait for stack analysis v2 to finish with user key
+
+    # Request ID check
+    Then I should find the external request id equals to id returned by stack analysis v2 post request
+
+    # Check all parameters of response including timestamp
+     And I should get stack analyses v2 response with all attributes
+
+    # SLA/SLO-related checks
+    When I look at the stack analysis duration
+    Then I should see that the duration is less than 180 seconds
+
+    # Analyzed dependencies checks
+    When I look at recent stack analysis
+    Then I should find <analyzed_deps_count> analyzed dependencies for stack analyses v2
+
+    # Unknown dependencies checks
+    When I look at recent stack analysis
+    Then I should find <unknown_deps_count> unknown dependencies for stack analyses v2
+
+    # Check license count
+    When I look at recent stack analysis
+    Then I should find <license_count> total licenses for stack analyses v2
+     And I should find <unknown_license_count> unknown licenses for stack analyses v2
+     And I should find <distinct_license_count> distinct licenses for stack analyses v2
+
+    # Verify recommendation data
+    When I look at recent stack analysis
+    Then I should find <usage_outlier_count> usage outliers for stack analyses v2
+     And I should find <companion_count> companions for stack analyses v2
+
+    # Verify Free user having registration link.
+    When I look at recent stack analysis
+    Then I should find registration link for stack analyses v2
+
+    # Analyzed dependencies attribute check
+    When I look at recent stack analysis
+    Then I should find all attribute about analyzed dependencies for stack analyses v2
+
+     Examples: files containing packages to test
+     | ecosystem | file                         | analyzed_deps_count | unknown_deps_count | license_count | unknown_license_count | distinct_license_count | usage_outlier_count | companion_count |
```

Comment:
these were the same files v1 was using can you add different files with newer dependencies from snyk feed , and also files with same packages and different versions is not required , just keep one best one , it will also decrease the e2e run time